### PR TITLE
reorderd the attributes in accordance with the style guide

### PR DESCRIPTION
**Pull request by demcg**
reorderd the attributes in accordance with the style guide and adjusted the spacing at the end of attribute lists.

### CODE REVIEW

File: vip_spec.xsd [hunk 1]
```diff
@@ -372,9 +372,9 @@ reference elements for polling location
                         </xs:sequence>
                       </xs:complexType>
                     </xs:element>
-                    <xs:element maxOccurs="1" type="xs:boolean" name="or_by_appointment" />
-                    <xs:element maxOccurs="1" type="xs:boolean" name="only_by_appointment" />
-                    <xs:element minOccurs="0" maxOccurs="1" type="xs:boolean" name="subject_to_change" />
+                    <xs:element type="xs:boolean" name="or_by_appointment" maxOccurs="1" />
+                    <xs:element type="xs:boolean" name="only_by_appointment" maxOccurs="1" />
+                    <xs:element type="xs:boolean" name="subject_to_change" minOccurs="0" maxOccurs="1" />
                   </xs:sequence>
```
Comment (cjerdonek) [inline vip_spec.xsd]: Name / type here.

**Comment by cjerdonek** [comment]
LGTM aside from the one comment.  (It would be nice to have a linter script for this!)

**Comment by demcg** [comment]
Thanks @cjerdonek

**Comment by jungshadow** [comment]
LGTM

**Comment by demcg** [comment]
@jungshadow this change is going to cause a great deal of diff noise and conflicts for changes made on long running branches. Are any of the existing feature branches that touch vip_spec.xsd going to merged, or are they no longer active?

**Comment by jungshadow** [comment]
@demcg Hmm...there's one on there that I thought I had deleted, but I believe most are inactive/merged. @cjerdonek @pstenbjorn Can you delete any branches that you know are complete/out-of-date? I'll do the same.

**Comment by cjerdonek** [comment]
@jungshadow done.

**Comment by jungshadow** [comment]
My feeling is: merge this in, @demcg :) @pstenbjorn @cjerdonek Feel free to throw up a red flag if you think this should wait.

**Comment by cjerdonek** [comment]
Yep, do it! :grinning:

**Pull request merged**